cli-commands/commands.js

```diff
@@ -7,25 +7,38 @@ const test = require('./etherlime-test/test');
 
 const commands = [
 	{
-		command: 'ganache [port]',
+		command: 'ganache [port] [output]',
 		description: 'start etherlime ganache-cli instance with static accounts with a lot of ETH.',
 		argumentsProcessor: (yargs) => {
 			yargs.positional('port', {
 				describe: 'port to run ganache-cli on',
 				type: 'number'
-			})
+			});
+
+			yargs.positional('output', {
+				describe: 'Defines the way that the logs are shown',
+				type: 'string',
+				default: 'normal',
+				choices: ['none', 'normal', 'structured']
+			});
 		},
 		commandProcessor: (argv) => {
-			ganache.run(argv.port, console);
+			ganache.run(argv.port, argv.output, console);
 		}
 	},
 	{
-		command: 'init',
+		command: 'init [output]',
 		description: 'initialize deployment folder structure and deployment files ready for etherlime deploy',
 		argumentsProcessor: (yargs) => {
+			yargs.positional('output', {
+				describe: 'Defines the way that the logs are shown',
+				type: 'string',
+				default: 'normal',
+				choices: ['none', 'normal', 'structured']
+			});
 		},
 		commandProcessor: async (argv) => {
-			await init.run();
+			await init.run(argv.output);
```

Comment:
As an alternative idea, wouldn't it be easier for you, instead of passing this in every function, and adding tons of logic for when to store and when to remove the output file, to add this logic in the commandProcessor. Something like this:
```
logger.storeOutputParameter(output);
await init.run();
logger.removeOutputStorage();
```

Reply:
This will also help with your problem, of one command overwriting the output file of another